src/azure-cli/azure/cli/command_modules/ams/operations/mru.py

```diff
@@ -16,12 +16,24 @@
              2: 'S3'}
 
 
-def get_mru(cmd, resource_group_name, account_name):
+def get_mru(client, cmd, resource_group_name, account_name):
+    account_info = client.get(resource_group_name,
+                              account_name) if resource_group_name else client.get_by_subscription(account_name)
+    if account_info.encryption:
+        raise CLIError('The media reserved unit operation failed as the Media Services account was created with the'
```

Comment:
Please use a detailed error type from [azclierror](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/azclierror.py) according to the [error handling guide](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md#available-error-types).